### PR TITLE
Fixes site organization and several other things (details below). 

### PR DESCRIPTION
**Pull request by austinchambers**
1. Fixes site organization. It's now much simpler. Intro page shows basic info for preflight and support-bundle. Click "Getting Started" in either to be taken to the relevant section. Docs are organized under either preflight or support-bundle, not both. This also obviated the need for breadcrumbs, which I removed. Ensured all old aliases work. 
2. Fixed title of reference page (should be "Reference Documentation" not "Vendor Documentation")
3. Some CI fixes (it was complaining when you'd add a new page. Fixed by using relative URLs and manually defining the structure of our alias html). 
4. Added highlighting, so that the current section of the page you're in is highlighted in the main menu. This also required some CSS fixes. 
5. Removed obsolete/incorrect code. A little bit of cleanup. 

image

image

image

### CODE REVIEW

File: themes/hugo-whisper-theme/layouts/index.html [hunk 1]
```diff
@@ -32,34 +32,52 @@
...
+              <div class="row">&nbsp;</div>
+              <div class="row button-wrapper">
+                <a class="button primary-button" href="/preflight/overview/">
+                  Get started
+                </a>
+                <a class="button border-button" href="https://github.com/replicatedhq/troubleshoot/releases" target="_blank">
+                  Download latest version
+                </a>
+              </div>
+              <div class="row"><p style="margin-bottom:0px; font-size:16px;">Install on your workstation:</p></div>
```
Comment (GraysonNull) [inline themes/hugo-whisper-theme/layouts/index.html]: Can we add 12px of margin-bottom here?

File: themes/hugo-whisper-theme/layouts/index.html [hunk 2]
```diff
@@ -32,34 +32,52 @@
...
+                <div class="row">&nbsp;</div>
+                <div class="row button-wrapper">
+                  <a class="button primary-button" href="/support-bundle/overview/">
+                    Get started
+                  </a>
+                  <a class="button border-button" href="https://github.com/replicatedhq/troubleshoot/releases" target="_blank">
+                    Download latest version
+                  </a>
+                </div>
+                <div class="row"><p style="margin-bottom:0px; font-size:16px;">Install on your workstation:</p></div>
```
Comment (GraysonNull) [inline themes/hugo-whisper-theme/layouts/index.html]: Same here

Comment (austinchambers) [reply themes/hugo-whisper-theme/layouts/index.html]: Fixed

File: themes/hugo-whisper-theme/layouts/index.html (hunk 1 above)
Comment (austinchambers) [reply themes/hugo-whisper-theme/layouts/index.html]: Fixed

**Pull request merged**